src/processor/operator/persistent/reader/csv/parallel_csv_reader.cpp

```diff
@@ -156,6 +156,10 @@ static offset_t tableFunc(TableFuncInput& input, TableFuncOutput& output) {
 static std::unique_ptr<TableFuncBindData> bindFunc(main::ClientContext* /*context*/,
     TableFuncBindInput* input) {
     auto scanInput = ku_dynamic_cast<TableFuncBindInput*, ScanTableFuncBindInput*>(input);
+    if (scanInput->expectedColumnTypes.size() > 0) {
+        scanInput->config.options.insert_or_assign("SAMPLE_SIZE",
+            Value((int64_t)0)); // only scan headers
```

Comment:
Should sample size be a positive integer?
The value should be uint64 instead of int64

Reply:
All integer parameters should be `INT64` so that csvconfig can detect and parse them.